setup.cfg

```diff
@@ -1,5 +1,6 @@
 [pep8]
 max-line-length=79
+ignore=E225,E226
```

Comment:
I'm in doubt that we should ignore these. Any reasons why to disable them globally?

Reply:
Well, I'm not in place to judge this. I did it to be able to save a file which has `64*1000` literal without spaces around `*`. Otherwise autopep just fixes it for me.

Reply:
That autopep suggestion looks correct. You may also write `64000` instead without any loss in readability.

Reply:
I'm with you.  @asvetlov specifically asked me to keep it as was.

Reply:
For me the strange thing is successful passing flake8 checks without the `ignore` setting.
I'm totally fine with 64000 or even better 0x10000. Pretty sure it should be 64KiB instead of 64kB

Reply:
Please drop the change. Feel free to replace the limit with `0x10000` if needed -- I'm +-0 for it.

Reply:
done